src/ApparentHorizons/SpherepackIterator.hpp

```diff
@@ -0,0 +1,115 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+///
+/// \file
+/// Defines class SpherepackIterator.
+
+#pragma once
+
+#include "ErrorHandling/Assert.hpp"
```

Comment:
For consistency: elsewhere we place system headers first, then blank line, then SpECTRE headers.